deny.toml

```diff
@@ -42,11 +43,21 @@ allow = [
     "Apache-2.0",
     "BSD-2-Clause",
     "BSD-3-Clause",
+    "BSL-1.0", # Boost Software License 1.0
     "CC0-1.0",
+    "ISC",
     "MIT",
-    "BSL-1.0", # Boost Software License 1.0
-    "Unicode-DFS-2016",
     "MPL-2.0",
+    "OpenSSL",
+    "Unicode-DFS-2016",
+    "Zlib",
```

Comment:
These entries are now alphabetically sorted, new entries are: ISC, OpenSSL and Zlib